models/index.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('Sequelize')
+
+const MoviesModels = require('./movies')
+
+const connection = new Sequelize('movies', 'movies', 'MOVIES', {
```

Comment:
This config should be dynamic using the same config file your migrations would use.